requirements-dev-arrow.txt

```diff
@@ -1,2 +1,2 @@
 feather-format>=0.4.1
-pyarrow<3.1.0
+pyarrow
```

Comment:
Should we leave in a minimum version?

Reply:
I'm not sure what the minimum version should be. It looks pyarrow 3.1.0 if you can install from a wheel (ie in our CI) but takes some work if you are trying to build from source.